packages/odyssey-react/src/components/Button/index.tsx

```diff
@@ -25,7 +25,7 @@ export type Props = {
   /**
    * Text content to be rendered within the button, usualy label text.
    */
-  children: ReactText;
+  children: ReactNode;
```

Comment:
why the change to the default type here for `children` ? I think we should be able to remove this entirely given the right type of refactoring to simplify types